cmd/sqlcmd/sqlcmd.go

```diff
@@ -126,14 +126,21 @@ func Execute(version string) {
 		PreRunE: func(cmd *cobra.Command, argss []string) error {
 			SetScreenWidthFlag(&args, cmd)
 			if err := args.Validate(); err != nil {
+				cmd.SilenceUsage = true
 				return err
 			}
 			if err := normalizeFlags(cmd); err != nil {
+				cmd.SilenceUsage = true
 				return err
 			}
 			return nil
 		},
 		Run: func(cmd *cobra.Command, argss []string) {
+			if len(argss) > 0 {
+				fmt.Printf("Sqlcmd: '%s': Unknown Option. Enter '-?' for help.", argss[0])
```

Comment:
Let's make the text "Unknown command.  Enter '--help' for command help.".   (do this because, if the user does end a bad flag, e.g. --badflag, then we do give the user the flag help.  But in this case the user is entering a bad command, e.g. sqlcmd badbad, so they probably want the sqlcmd --help text.